test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1014,109 +1016,6 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 		})
 
-		It("should be able to scale an iis webserver", func() {
```

Comment:
Should these test cases be removed?

Reply:
yes, this test was replaced with a more comprehensive one